### DISCUSSION TITLE
"Héðan í frá má enginn hunsa neinn" - 2024/6/4

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#279] [vefsida-ummæli]
# "Héðan í frá má enginn hunsa neinn" - 2024/6/4

Heimasíða Biskupsreglunnar

https://www.biskupsreglan.com/frettir/Enginn-m%C3%A1-hunsa

**Comment by TJ478** [comment]
Góð hugmynd
